domain-models-interface-extensions/src/main/java/org/folio/rest/tools/utils/Envs.java

```diff
@@ -12,7 +12,8 @@
   DB_DATABASE,
   DB_QUERYTIMEOUT,
   DB_CHARSET,
-  DB_MAXPOOLSIZE;
+  DB_MAXPOOLSIZE,
+  RMB_EXPLAIN_QUERY_THRESHOLD;
```

Comment:
To make it work in Envs it must start with `DB_`

Reply:
It was having prefix NOT DB_ to avoid it passing passed to the Mysql/postgres client.. (which would probably choke if we passed an unknown setting ). I don't have tests / or it-tests with the env-thingy set (since I don't know how to set System env in tests), but manual testing suggests it works fine.